docs/getting-started/install/linux/index.md

```diff
@@ -87,6 +90,8 @@ The package ***referenceassemblies-pcl*** should be installed for PCL compilatio
 
 The package ***ca-certificates-mono*** should be installed to get SSL certificates for HTTPS connections. Install this package if you run into trouble making HTTPS connections.
 
+The package ***msbuild*** may be needed to compile very new projects which are not compatible with the deprecated MSBuild replacement, XBuild.
```

Comment:
Given that we need a story for people doing `./configure; make; make install` from the tarball that might be a good alternative because they'll be screwed otherwise once we fully deprecate xbuild (unless they somehow find out how to download+build msbuild)